.gitignore

```diff
@@ -116,3 +116,6 @@ appsettings.json
 local.settings.json
 .python_packages
 beta-feedback-api.zip
+
+# ignore reports created by report_generator
+feedback_report.csv
```

Comment:
I would add env.sh to .gitignore to prevent environment variables being pushed to github

Reply:
There's already a file called setup_env_vars.sh I added to .gitignore of this repo, so might as well go with that.